PC/winreg.c

```diff
@@ -1398,6 +1416,8 @@ winreg.OpenKey -> HKEY
     access: REGSAM(c_default='KEY_READ') = winreg.KEY_READ
         An integer that specifies an access mask that describes the desired
         security access for the key.  Default is KEY_READ.
+    options: int = 0
+        Can be one of the REG_OPTION_* constants.
```

Comment:
We should probably document the interaction with `reserved` here. I'm not sure what the history is of `reserved` becoming a meaningful parameter, but it's meaningful now, so we should say that here (and in the real docs, `Doc\library\winreg.rst`). (I mean "here" as in a few lines above, but I can't comment on it because it hasn't been updated yet).